Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -37,6 +38,29 @@ extension BrowserViewController {
         popover.addsConvenientDismissalMargins = false
         popover.present(from: topToolbar.locationView.rewardsButton, on: self)
     }
+    
+    func configureRewardsLogs() {
+        if rewards == nil { return }
+        RewardsLogger.configure(logCallback: logCallback, withFlush: nil)
+    }
+    
+    private func logCallback(logLevel: LogLevel, line: Int32, file: String, data: String) {
```

Comment:
Should this function be static? How does `logCallback` get called when it's non-static? Does self somehow get retained with the block when we do `RewardsLogger.configure`?

Just curious is all.

Anyway, code looks good!

Reply:
Thats a good point, not sure that this being non-static may make it hold a ref to `BrowserViewController` or if its copying it as a block somehow… It might not be a huge issue anyways since we never dealloc BrowserViewController :P

Reply:
This function doesn't call BrowserViewController so we should be fine